webui/react/src/services/apiBuilder.ts

```diff
@@ -38,15 +38,15 @@ export function generateApi<Input, Output>(api: Api<Input, Output>) {
       return api.postProcess ? api.postProcess(response) : response.data as Output;
     } catch (e) {
       const isAuthError = isAuthFailure(e);
-      const error = handleError({
+      handleError({
         error: e,
         level: isAuthError ? ErrorLevel.Fatal : ErrorLevel.Error,
         message: isAuthError ?
           `unauthenticated request ${api.name}` : `request ${api.name} failed.`,
         silent: true,
         type: isAuthError ? ErrorType.Auth : ErrorType.Server,
       });
-      throw error;
+      throw e;
```

Comment:
this change is here so the callers still can detect that error is from request cancellation. unless we introduce that as an error type maybe?